### DISCUSSION TITLE
pyinstaller not supporting dataframe_image library

### DISCUSSION THREAD

**Discussion by sophia-xl** [pyinstaller/pyinstaller#6432] [Features & Development]
Hi, I was wondering if the dataframe_image package is supported in pyinstaller, and if not, if there's a way to request for an addition? Thank you!

**Comment by bwoodsend** [comment]
If it doesn't work and you can create a minimal example that shows it not working then you can [request it over on the hooks repo](https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/new?assignees=&labels=good+first+issue%2C+hook-request&template=new-hook-request.md&title=).

**Reply by sophia-xl** [reply]
I'll try this, thank you!

**Comment by rokm** [comment]
Depending on configuration, this package uses either system-installed Chrome or system-installed `latex` environment (with several packages that may not be installed by default). So with either option, the concept of self-contained frozen application goes out of window.

FWIW, the program using `dataframe_image` with `latex` backend seems to work on linux as long as `jupyter` templates are collected. The problem is those are installed outside of `site-packages`, so they need to be collected manually, either writing a proper hook, or via command-line `--add-data` switch. For example, on linux: `--add-data "venv/share/jupyter:share/jupyter"`.

No idea what requirements for Chrome backend are, though...

**Reply by sophia-xl** [reply]
I'll also try this, thank you!

**Comment by wodns** [comment]
@sophia-xl
If you have problem with `FileNotFoundError: [Errno 2] No such file or directory: dataframe_image\\static\\style.css`,
see https://github.com/dexplo/dataframe_image/issues/32.

You can add `('venv/Lib/site-packages/dataframe_image/static/style.css', 'dataframe_image/static')` to `datas` in `.spec` file
or
via command line `--add-data=venv/Lib/site-packages/dataframe_image/static/style.css;dataframe_image/static`
